### DISCUSSION TITLE
ospsuite-R add event

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1876] [Questions & Problems] [answered]
Hi all,
I would like to ask if ospsuite-R now supports adding additional “Events”. Or is it possible to add/clone an existing “application” to an “Event” as shown below:
![image](https://github.com/user-attachments/assets/dcdd6087-973c-4d99-9c1e-9ff00983d6e5) 
![image](https://github.com/user-attachments/assets/94b8f5cd-2d14-40be-9254-df4c172c3f29)

**Comment by PavelBal** [comment, marked as answer]
Unfortunately, this is not possible from R right now. The possibility to dynamically create a series of events is planned for version 13.

**Reply by SCZ1174** [reply]
The current official version of R seems to be Version 11.3.520, and version 13 still feels a long way off, haha. But really looking forward to it!

**Comment by PavelBal** [comment]
Version 12 will be released very soon, after that we will full start with with the development of v13.
